Makefile

```diff
@@ -19,4 +19,4 @@ test:
 	phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
 
 docs:
-	cd Resources/doc && sphinx-build -W -b html -d _build/doctrees . _build/html
+	cd docs && sphinx-build -W -b html -d _build/doctrees . _build/html
```

Comment:
You still have that fix, and should not I think.

Reply:
well, even after I had merged the master, it was failing. 

I tried several times without success. dont know why it was like that.

Reply:
https://travis-ci.org/sonata-project/SonataNotificationBundle/jobs/303040587

Reply:
Oh it's another file as in my PR!

Reply:
Then leave it like that, it's fine.

Reply:
The strange thing is. why the tests passed in your fix branch?

Reply:
Because I did not change any docs 😓

Reply:
got it